docs/GettingStarted.md

```diff
@@ -3,47 +3,252 @@
+```
+
+* Publish the functions to the function app
+
+```bash
+func azure functionapp publish ${tpfunctionapp} --dotnet
+```
+
+### Deploy the fleet integration layer
+
+In this step you will deploy the resources required for the fleet integration layer
+
+#### Execute fleet integration biceps deployment scripts
+
+- Change your directory to ```infra/deployment/FleetIntegration```
 
-- Follow the instructions to deploy the infrastructure for each layer.
-  - [Telemetry Platform](../infra/deployment/TelemetryPlatform/README.md) requires creating test certificates andcreating Azure resources for Event Grid, Event Grid Topics, Azure Functions and test devices.
-  - [Fleet Integration](../infra/deployment/FleetIntegration/README.md) deploy the resources to process messages into analytics and business integration.
+- [Create a resource group](https://learn.microsoft.com/cli/azure/manage-azure-groups-azure-cli#create-a-resource-group) for the deployment in a region
 
-- Deploy and configure the Azure Functions
-  - [Deploy and configure Telemetry Platform](../src/TelemetryPlatform/Functions/README.md) deploys and configures the message processing code
-  - [Deploy and configure Fleet Integration](../src/FleetIntegration/Functions/README.md) deploys and configures the dataverse integration code
+``` bash
+az group create --name ${RG_FLEETINTEGRATION} --location ${LOCATION}
+```
+
+- Get the name of the event hub created in the telemetry platform.
+
+``` bash
+export tpeventhubname=$(az eventhubs namespace list --resource-group ${RG_TELEMETRYPLATFORM} --query "[].name" --output tsv)
+```
+
+- Execute the main.bicep refering to your resource group and using the event hub namespace from the telemetry platform as an argument.
+
+``` bash
+az deployment group create --resource-group ${RG_FLEETINTEGRATION} --template-file ./main.bicep --parameters evhnsTelemetryPlatformNamespaceName=${tpeventhubname}
+```
+
+You can try out sending messages and the integration with Azure Data Explorer.
+
+#### Deploy the telemetry platform functions for connection to the dataverse
+
+> [!NOTE]
+> This step requires a Dynamics 365 installation with Field Service. You can skip this step in case you don't have an installation available.
+
+Follow the instructions in [Dataverse Integration](DataverseIntegration.md) to automatically create ```IoT Event``` entities.
+
+## Try it out
+
+### Send sample messages
+
+The MQTT Test Client will send sample payloads using the generated certificates. The client will create 5 connections to the MQTT broker functionality
+of Event Grid, and send a JSON payload every second per device.
+
+The payloads can be found in the ./SamplePayloads directory. It consists of a JSON array that contains several values, harmonized using the
+Vehicle Signal Specification (VSS) from COVESA.
+
+Change your directory to ```./src/TestClient``` and build the client
+
+```bash
+dotnet build
```

Comment:
This build step emitted a bunch of warnings. I think they all look like easy fixes. Are you able to fix those up please?